CHANGELOG.md

```diff
@@ -1,3 +1,243 @@
+
+* Fixes a bug that could cause re-partition joins involving local shards to fail
+
+* Fixes a bug that limits usage of sequences in non-int columns
+
+* Fixes a bug that prevents `DROP SCHEMA CASCADE`
+
+* Fixes a clog lookup failure that could occur when writing to a columnar table
+
+* Fixes a crash that occurs when the aggregate that cannot be pushed-down
+  returns empty result from a worker
+
+* Fixes a missing `FROM` clause entry error
+
+* Fixes a possible segfault that could happen when reporting distributed
+  deadlock
+
+* Fixes an issue that could cause unexpected errors when there is an in-progress
+  write to a columnar table
+
+* Fixes an unexpected error that occurs due to aborted writes to a columnar
+  table with an index
+
+* Fixes an unexpected error that occurs when writing to a columnar table created
+  in older version
+
+* Fixes function signature generation bug
+
+* Fixes issue when compiling Citus from source with some compilers
+
+* Fixes issues on `ATTACH PARTITION` logic
+
+* Fixes naming issues of newly created partitioned indexes
+
+* Implements `TEXT SEARCH CONFIGURATION` propagation
+
+* Implements `TEXT SEARCH DICTIONARY` propagation
+
+* Improves concurrent metadata syncing and metadata changing DDL operations
+
+* Improves self-deadlock prevention for `CREATE INDEX / REINDEX CONCURRENTLY`
+  commands for builds using PG14 or higher
+
+* Introduces `citus_backend_gpid()`
+
+* Introduces `citus_check_cluster_node_health` UDF to check cluster connectivity
+
+* Introduces `citus_check_connection_to_node` UDF to check node connectivity
+
+* Introduces `citus_coordinator_nodeid` UDF
+
+* Introduces `citus_stat_activity` view
+
+* Introduces `use_citus_managed_tables` GUC
+
+* Introduces a new flag `force_delegation` in `create_distributed_function()`,
+  which will allow a function to be delegated in an explicit transaction block.
+  Such delegated functions are restricted to use only the distributed argument
+  value.
+
+* Introduces global pid
+
+* Introduces helper functions for global pid
+
+* Moves `pg_dist_object` to `pg_catalog` schema
+
+* Overrides `pg_cancel_backend()` and `pg_terminate_backend()` to run with
+  global pid
+
+* Partitions shards to be co-located with the parent shards
+
+* Prevents Citus table functions from being called on shards
+
+* Prevents creating distributed functions when there are out of sync nodes
+
+* Prevents failing over to local execution for DDLs that cannot be executed
+  locally
+
+* Propagates `CREATE SCHEMA .. GRANT ON SCHEMA ..` commands
+
+* Propagates `pg_dist_object` to worker nodes
+
+* Propagates schema operations
+
+* Propagates tables and sequences like other objects to remote nodes
+
+* Provides notice message for idempotent `create_distributed_function` calls
+
+* Pushes down procedures with OUT arguments to the worker nodes
+
+* Refactor `GenerateGrantOnSchemaStmtForRights` to a more generic form
+
+* Reinstates optimisation for uniform shard interval ranges
```

Comment:
this seems not an interesting user facing feature, maybe drop?

Reply:
we had this entry for older releases, so might be nice to keep for consistency ?

Reply:
Ah, that's tricky. Then, lets keep